#This represents the environment information

Header header

#task infor
int32 task_0_pose_ID
int32 task_0_infor_value
int32 task_1_pose_ID
int32 task_1_infor_value
int32 task_2_pose_ID
int32 task_2_infor_value
int32 task_3_pose_ID
int32 task_3_infor_value

#threat value
int32 threat_0_pose_ID
int32 threat_0_value
int32 threat_1_pose_ID
int32 threat_1_value
int32 threat_2_pose_ID
int32 threat_2_value
int32 threat_3_pose_ID
int32 threat_3_value
int32 threat_4_pose_ID
int32 threat_4_value


#edge jam value
int32 jam_edge_0_pose_ID
int32 jam_edge_0_value
int32 jam_edge_1_pose_ID
int32 jam_edge_1_value
int32 jam_edge_2_pose_ID
int32 jam_edge_2_value
int32 jam_edge_3_pose_ID
int32 jam_edge_3_value
int32 jam_edge_4_pose_ID
int32 jam_edge_4_value
int32 jam_edge_5_pose_ID
int32 jam_edge_5_value


